tests/sqlparser_redshift.rs

```diff
@@ -353,3 +378,20 @@ fn test_parse_json_path_from() {
         _ => panic!(),
     }
 }
+
+#[test]
+fn test_parse_select_numbered_columns() {
+    redshift_and_generic().verified_stmt(r#"SELECT 1 AS "1" FROM a"#);
+    // RedShift specific case - quoted identifier inside square bracket
```

Comment:
```suggestion
```
this one seems to be already covered by `test_parse_create_numbered_columns`?

Reply:
I split the test into two to make it more explicit:

1. `test_parse_select_numbered_columns` - it is a reason for this MR
2. `test_parse_nested_quoted_identifier`